### ISSUE TITLE
defer always rendered for scripts in src/Asset/TagRenderer.php

### ISSUE THREAD

**Issue by lamselli** [symfony/webpack-encore-bundle#103]
In last version, I didn't want my scripts to have "defer" attribute, as I was using old jquery calls inside my rendered page.
But the defer attribute is added even when you set defer to false in config file (webpack_encore.yaml)
`script_attributes:
    defer: false`
    
As a workaround, I had to create a subscriber event to RenderAssetTagEvent to remove the attribute defer if its value is equal to false.

Sorry if I am unclear, but english is not my native language.

**Comment by maxhelias** [comment]
Maybe an `array_filter` with a condition on the null value here : https://github.com/symfony/webpack-encore-bundle/blob/main/src/Asset/TagRenderer.php#L183, could be the solution.

**Comment by weaverryan** [comment]
Fixed in #113. But also, even before that version, you can remove the `defer` attribute from the config :)

```diff
webpack_encore:
    script_attributes:
-        defer: true
```

Cheers!
